### PR TITLE
Move SimpleMessageFormatter from yiisoft/translator-formatter-simple

### PR DESCRIPTION
**Pull request by DAGpro**
| Q             | A
| ------------- | ---
| New feature?  | ✔️
| Breaks BC?    | ❌
|  issues  |  yiisoft/translator-formatter-simple#25

### CODE REVIEW

File: config/common.php [hunk 1]
```diff
@@ -33,4 +35,6 @@
             $this->setLocale($params['yiisoft/translator']['locale']);
         },
     ],
+
+    MessageFormatterInterface::class => SimpleMessageFormatter::class,
```
Comment (xepozz) [inline config/common.php:39]: It will cause an error if you install `intl` formatter.
Could you set it up in the translator definition?

Comment (DAGpro) [reply config/common.php]: that's right ?  `MessageFormatterInterface::class => ['class' => SimpleMessageFormatter::class]`

Comment (xepozz) [reply config/common.php]: Nope. It's the same.

You need reconfigure `Translator` like the following snippet and remove current line.
image

Comment (DAGpro) [reply config/common.php]: so okay?

**Review by xepozz — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## 1.0.3 under development
 
-- no changes in this release.
+- Chg #25: Move SimpleMessageFormatter from package `yiisoft/translator-formatter-simple` in `yiisoft/translator` (@DAGpro, @vjik)
```
Comment (arogachev) [inline CHANGELOG.md:5]:
```suggestion
- Chg #25: Move `SimpleMessageFormatter` from `yiisoft/translator-formatter-simple` package (@DAGpro, @vjik)
```

**Review by arogachev — APPROVED** [review]

**Pull request merged**